File: CS/WinForms.Client/AuthForm.cs
using System;
using DevExpress.XtraEditors;

namespace WinForms.Client {
    public partial class AuthForm : XtraForm {
        public AuthForm() {
            InitializeComponent();
        }

        public string Login {
            get { return userNameEdit.Text; }
        }

        public string Password {
            get { return passwordEdit.Text; }
        }
    }
}

File: VB/WinForms.Client/AuthForm.vb
Imports DevExpress.XtraEditors

Namespace WinForms.Client

    Public Partial Class AuthForm
        Inherits XtraForm

        Public Sub New()
            InitializeComponent()
        End Sub

        Public ReadOnly Property Login As String
            Get
                Return userNameEdit.Text
            End Get
        End Property

        Public ReadOnly Property Password As String
            Get
                Return passwordEdit.Text
            End Get
        End Property
    End Class
End Namespace
